bundle/regal/lsp/completion/providers/locals/locals.rego

```diff
@@ -43,21 +43,15 @@ items contains item if {
 _excluded(line, position) if _function_args_position(substring(line, 0, position.character))
 
 _function_args_position(text) if {
-	text == trim_left(text, " \t")
 	contains(text, "(")
 	not contains(text, "=")
+	text == trim_left(text, " \t")
 }
 
 default _same_line_loop_vars(_) := []
 
-_same_line_loop_vars(line) := d if {
-	expr := trim_space(line)
-
-	strings.any_prefix_match(expr, {"some", "every"})
+_same_line_loop_vars(line) := vars if {
+	regex.match(`^\s*(some|every)`, line)
 
-	# ---------------------------------------------------> "some k, v in coll"
-	a := substring(expr, 0, indexof(expr, " in")) # -----> "some k, v"
-	b := trim_prefix(trim_prefix(a, "some"), "every") # -> "k, v"
-	c := replace(b, " ", "") # --------------------------> "k,v"
-	d := split(c, ",") # --------------------------------> [k, v] or [v]
+	vars := split(regex.replace(line, `(?:\s*(?:some|every)\s+)(\w+)(?:,?\s*)(\w+)?(?:\s+.*)`, "$1,$2"), ",")
```

Comment:
What if there is only one var? E.g. Wouldn't  `some x in foo` output `["x", "in"]`? 
Would this be expected, or we only end up here if we have two output vars?